### PR TITLE
Fix spelling mistake

### LINKED ISSUE #1: Spelling mistake
**Describe the bug**

There is a spelling mistake in the text

**To Reproduce**
1. Run the program and read the output
  
**Expected behavior**
"Helo" seems mispelled. I guess "Hello" would be better

**Screenshots**
not applicable

**Desktop (please complete the following information):**
 - OS: MacOS

**Smartphone (please complete the following information):**
not applicable

**Additional context**
not applicable

### PR DESCRIPTION
**Pull request by derlin**
# Description

I fixed the spelling mistake

Fixes #1

## Type of change

Please delete options that are not relevant.

- [ x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Visually.

**Test Configuration**:
* Python version:
* Operating System:

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 def main():
-    print("Helo World")
+    print("Hello World")
```
Comment (supcik) [inline hello.py]: Thank you

Comment (supcik) [inline hello.py]: What about a exclamation mark at the end ?

**Review by supcik — CHANGES_REQUESTED** [review]

File: hello.py (hunk 1 above)
Comment (derlin) [reply hello.py]: you're welcome

Comment (derlin) [reply hello.py]: nice, why not ?

**Review by supcik — APPROVED** [review]
Thank you

**Pull request merged**